test/unit/specs/components/VHeader/v-filter-button.spec.js

```diff
@@ -47,10 +48,13 @@ describe('VFilterButton', () => {
     })
     options = {
       localVue,
-      propsData: {
-        testContext: {
-          store: storeMock,
-          i18n,
+      propsData: props,
+      mocks: {
+        $nuxt: {
+          context: {
+            store: storeMock,
+            i18n,
+          },
         },
       },
```

Comment:
Did this work for you btw? I tried this to mock the `i18n` object and I could _not_ get it to work at all! That's why i ended up creating the `useI18n` composable so that it would be easier to mock.

Reply:
I just confirm, @sarayourfriend , and yes, it does work (I added some console.logs and ran the tests to be sure ) 
Did you also create the i18n object and load the messages as I did at the top of this file?
https://github.com/WordPress/openverse-frontend/pull/489/files/87c23926df95a8bfe4012069674e7485c12c5c18..e7f624fef9d52d8f279490cfd92ed6c96b18f6cb#diff-323a77061c65693ad59cc4cb90840d5fbcdc403428f023ffb3b4a448fc234a00R6-R31

Reply:
I did! I remember everything you've done here but I must have left _something_ out becuase I just could not get it to work! Glad you figured it out though 🎉

Reply:
It's weird, I really wonder why id didn't work now ... 🤔